our_site/planner/templates/planner/event_home_owned.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
Or what would be even better is if we could put a conditional in the html so that it displays one thing slightly differently depending on who the user is.